.github/workflows/go_wasm_build_test_deploy.yml

```diff
@@ -9,29 +9,80 @@ on:
   pull_request:
     branches: [ "main" ]
 
+env:
+  GO_VERSION: '1.22.5'
+  CHROME_VERSION: '129'
+  FIREFOX_VERSION: '131.0.3'
+  CHROME_DRIVER_VERSION: '129.0.6668.100'
+  GECKO_DRIVER_VERSION: '0.35.0'
+
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false # https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
     permissions:
       # Give the default GITHUB_TOKEN write permission to commit and push the
       # added or changed files to the repository.
       contents: write
     steps:
     - uses: actions/checkout@v4
 
+    - name: Testing git clone
+      shell: bash
+      run: ls
+
     - name: Set up Go
       uses: actions/setup-go@v4
       with:
-        go-version: '1.22.5'
+        go-version: ${{ env.GO_VERSION }}
 
     - name: Build
+      shell: bash
       run: ./build_wasm
 
-    - name: Test
-      run: ./run_tests
+    - name: Get WebDrivers
+      uses: actions/checkout@v4
+      with:
+        repository: 'padaiyal/depot'
+        path: ./depot/
```

Comment:
Is there an issue to change this to only pull required dependencies?

Reply:
Created to handle the required drivers: https://github.com/padaiyal/sanitizer/issues/49